crates/ruff/src/autofix/codemods.rs

```diff
@@ -11,6 +11,23 @@ use ruff_python_ast::source_code::{Locator, Stylist};
 use crate::cst::helpers::compose_module_path;
 use crate::cst::matchers::match_statement;
 
+/// Glue code to make libcst codegen work with ruff's Stylist
+pub(crate) trait CodegenStylist<'a>: Codegen<'a> {
+    fn codegen_stylist(&self, stylist: &'a Stylist<'a>) -> String;
```

Comment:
This seems to be sufficient (I honestly don't fully understand yet why lifetimes are required here)
```suggestion
    fn codegen_stylist(&self, stylist: &'a Stylist) -> String;
```